inc/camera_bonus.h

```diff
@@ -0,0 +1,188 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   camera_bonus.h                                     :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: sqiu <sqiu@student.42vienna.com>           +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/10/28 11:40:06 by u413q             #+#    #+#             */
+/*   Updated: 2024/01/14 17:38:52 by sqiu             ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef CAMERA_BONUS_H
+# define CAMERA_BONUS_H
+
+/* ====== LIBRARIES ====== */
+
+# include "utils_bonus.h"
+# include "miniRT_config_bonus.h"
+# include "colour_bonus.h"
+# include "entities_bonus.h"
```

Comment:
Colour and entities headers are required for the function:

t_colour	ft_anti_aliase_colour(int iterate[2], t_pixel_grid pixels, \
	t_cam cam, t_entities scene);

Otherwise it throws problems as it doesn't recognize both typedefs:
![image](https://github.com/u413-284-si/miniRT/assets/66411482/4f5b6ee6-b4d8-4ef5-979a-4e56e61b4317)

Weirdly enough it still compiles though.. would expect it to return an unknown type error. Would still keep both headers in cause this behaviour seems unreliable and not clean.

Reply:
Removed utils_bonus.h